products/jhipster.md

```diff
@@ -0,0 +1,70 @@
+---
+title: JHipster
+category: app
+versionCommand: yo jhipster
+permalink: /jhipster
+releasePolicyLink: https://www.jhipster.tech/releases/
+changelogTemplate: "https://github.com/jhipster/generator-jhipster/tag/__LATEST__"
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Supported
+#iconSlug: jhipster , WIP see https://github.com/jhipster/generator-jhipster/issues/20533
+
+#auto:
```

Comment:
I think auto configuration must be based on https://www.npmjs.com/package/generator-jhipster.

Reply:
:point_right: `auto` : I don't know how to achieve this

Reply:
Use something like 

    auto:
      - npm: https://www.npmjs.com/package/generator-jhipster

I just remembered NPM automation development is ongoing (https://github.com/endoflife-date/release-data/pull/36), so it will not work for the time being. But it does not hurt to configure it now.

Reply:
Done.